doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1065,11 +1065,12 @@ use cases of this functionality is the association of Context Providers.
 
 A context registration is represented by a JSON object with the following fields:
 
-+ `id`: Unique identifier assigned to the registration. Automatically created at creation time.
-+ `subject` : It s an object that describes the subject of the registration.
-+ `callback` : URL pointing to the service which is registered. In the case of a Context Provider
-corresponds to the URL of the provider service.
-+ `duration`: Duration of the registration in ISO8601 format. Default duration is infinite.
++ `id`       : Unique identifier assigned to the registration. Automatically generated at creation time.
++ `subject`  : It s an object that describes the subject of the registration.
++ `callback` : URL which denotes the end point of the registered service. It can be undefined under certain conditions,
+for instance, transient unavailability of the provider.
++ `metadata` : An optional dictionary which allows to associate extra metadata to the registration. 
```

Comment:
do we really need such a complexity, having that for this use case seems to be overkilling